### PR TITLE
Fix schedule update test failures

### PR DESCRIPTION
**Pull request by ecfm**
https://issues.cask.co/browse/CDAP-11522
https://builds.cask.co/browse/CDAP-DUT5749

### CODE REVIEW

**Comment by anew** [comment]
I think this is the correct fix - as long as we are still interacting with the two schedulers separately. LGTM if build passes, as a temp fix until the refactoring into a single scheduler API has happened.

**Comment by ecfm** [comment]
Build passed, merging

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/audit/AuditPublishTest.java [hunk 1]
```diff
@@ -74,6 +75,7 @@
   private static CConfiguration cConf;
   private static MessagingService messagingService;
   private static TopicId auditTopic;
+  private static Scheduler scheduler;
```
Comment (anwar6953) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/audit/AuditPublishTest.java]: unused?

**Comment by anwar6953** [comment]
1 minor comment. Remainder LGTM

**Pull request merged**